tests/model_registry/rest_api/utils.py

```diff
@@ -91,6 +105,38 @@ def register_model_rest_api(
     return {"register_model": register_model, "model_version": model_version, "model_artifact": model_artifact}
 
 
+def create_ca_bundle_file(
```

Comment:
Can you enhance https://github.com/opendatahub-io/opendatahub-tests/blob/main/utilities/certificates_utils.py#L21 or rename and move it to the same file?